templates/.walg.json.j2

```diff
@@ -0,0 +1,10 @@
+{
+  "AWS_ACCESS_KEY_ID": "{{ aws_access_key_id }}",
+  "AWS_ENDPOINT": "{{ aws_endpoint }}",
+  "AWS_S3_FORCE_PATH_STYLE": "true",
+  "AWS_SECRET_ACCESS_KEY": "{{ aws_secret_access_key }}",
+  "PGDATA": "{{ postgresql_data_dir }}",
+  "PGHOST": "{{ postgresql_unix_socket_dir }}",
+  "WALE_S3_PREFIX": "{{ wale_s3_prefix }}",
+  "WALG_COMPRESSION_METHOD": "brotli"
```

Comment:
I think we need a variable for "WALG_COMPRESSION_METHOD". 
Since a different compression method may be required (possible options are: lz4, 'lzma', 'brotli').